src/Bladeburner/Bladeburner.ts

```diff
@@ -750,32 +751,20 @@ export class Bladeburner {
     }
   }
 
+  public killRandomSupportingSleeves(n: number) {
```

Comment:
Isn't `public` the default? Does it need to be stated?

Reply:
You marked this as resolved, but nothing changed here, did you forget?

Reply:
Human error.